docs/api.md

```diff
@@ -377,7 +377,7 @@ ao.pInstrument(spanInfo, run).then(...)
 ```
 <a name="ao.startOrContinueTrace"></a>
 
-### ao.startOrContinueTrace(xtrace, span, runner, [opts], [callback]) ⇒ <code>value</code>
+### ao.startOrContinueTrace(traceparent, tracestat, span, runner, [opts], [callback]) ⇒ <code>value</code>
```

Comment:
Trivial: `tracestate` instead of `tracestat`

Reply:
Fixed.